specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscNodeConfiguration.json

```diff
@@ -0,0 +1,416 @@
+        },
+        "x-ms-examples": {
+          "Delete a DSC node configuration": {
+            "$ref": "./examples/deleteDscNodeConfiguration.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "nodeConfigurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Dsc node configuration name."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "DscNodeConfiguration"
+        ],
+        "operationId": "DscNodeConfiguration_Get",
+        "description": "Retrieve the Dsc node configurations by node configuration.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/dscnodeconfigurations"
+        },
+        "x-ms-examples": {
+          "Get a DSC node configuration": {
+            "$ref": "./examples/getDscNodeConfiguration.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "nodeConfigurationName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The Dsc node configuration name."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DscNodeConfiguration"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "DscNodeConfiguration"
+        ],
+        "operationId": "DscNodeConfiguration_CreateOrUpdate",
```

Comment:
>DscNodeConfiguration_CreateOrUpdate [](start = 24, length = 35)

Is this a long running operation?

Reply:
No. DscNodeconfigurations are usually very small in size. and most of the PUT calls use bloblink rather than upload from local files.

Reply:
Does the client have to poll?

Reply:
Yes

Reply:
Done.

Reply:
Fixed